README.md

```diff
@@ -2,29 +2,85 @@
 
 [![Build Status](https://github.com/findologic/plugin-shopware-6/workflows/PHPUnit/badge.svg?branch=main)](https://github.com/findologic/plugin-shopware-6/actions)
 
-## Table of Contents
```

Comment:
Table of contents is automatically generated by GitHub since a few months, so we no longer need to add our own.